src/main/java/org/scijava/ui/swing/script/autocompletion/JythonAutoCompletion.java

```diff
@@ -47,7 +47,8 @@ public JythonAutoCompletion(final CompletionProvider provider) {
 	}
 	
 	static private final Pattern importPattern = Pattern.compile("^(from[ \\t]+([a-zA-Z_][a-zA-Z0-9._]*)[ \\t]+|)import[ \\t]+([a-zA-Z_][a-zA-Z0-9_]*[ \\ta-zA-Z0-9_,]*)[ \\t]*([\\\\]*|)[  \\t]*(#.*|)$"),
-								 tripleQuotePattern = Pattern.compile("\"\"\"");
+								 tripleQuotePattern = Pattern.compile("\"\"\""),
+								 variableDeclarationPattern = Pattern.compile("([a-zA-Z_][a-zA-Z0-9._]*)\\s*=\\s*([A-Z_][a-zA-Z0-9._]*)(?:\\()"); // E.g., in 'imp=ImagePlus()' grou1: imp; group2: ImagePlus
```

Comment:
Not all kinds of whitespace are valid, for example a line break isn't valid if the line doesn't end with a backslash. That's why I used [ \\t] instead of \\s.

Reply:
Good point. will fix.